### DISCUSSION TITLE
PostrgeSQL support

### DISCUSSION THREAD

**Discussion by keith6014** [photoprism/photoprism#4352] [Ideas]
I recently start to use PhotoPrism. Its a great product. However, I am not familar with Mysql (MariaDB, Oracle product). I was wondering if we can get PostgreSQL support into PhotoPrism. I can volunteer my time to port it as long as I get some guidance from the core authors.

**Comment by lastzero** [comment]
We appreciate any contributions in this regard, see:
- https://github.com/photoprism/photoprism/issues/47#issuecomment-2185893061
